crates/camera/src/camera.rs

```diff
@@ -209,7 +209,7 @@ impl DesiredPoW {
     }
 }
 
-fn setup(mut commands: Commands) {
+fn setup(mut commands: Commands, camera_query: Query<Entity, With<Camera3d>>) {
```

Comment:
I will have to thing about it a bit more, but my current thought is that the whole map cycling & game setup has to be done differently:

* Proper clean up after `GameState::Playing` is existed has to be implemented.
* Map selection should be done via main menu. It might be very simplistic at this point (a button for each available map?).
* Generally: once you enter a game (`GameState::Playing`) you cannot change its configuration (like in most RTS games). You have to quit the game first and then start a new game.


My second point is of highest priority IMO.

Reply:
Related #34.